src/app/components/search/results/results.component.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ResultsComponent } from './results.component';
+import { CaseData } from '../../../types/case';
+import { HearingData } from 'src/app/types/hearing';
+
+describe('ResultsComponent', () => {
+  let component: ResultsComponent;
+  let fixture: ComponentFixture<ResultsComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [ResultsComponent],
+    });
+    fixture = TestBed.createComponent(ResultsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should return Multiple if there are multiple defendants or judge', () => {
+    spyOn(component, 'getNameValue').and.callThrough();
+    const def = ['Defendant 1', 'Defendant 2'];
+    const dValue = component.getNameValue(def);
+    const judges = ['Judge 1', 'Judge 2'];
+    const jValue = component.getNameValue(judges);
+
+    expect(component.getNameValue).toHaveBeenCalled();
+    expect(dValue).toBe('Multiple');
+    expect(jValue).toBe('Multiple');
+  });
+
+  it('should return single name if there is a single defendant or judge', () => {
+    spyOn(component, 'getNameValue').and.callThrough();
+    const def = ['Defendant 1'];
+    const dValue = component.getNameValue(def);
+    const judges = ['Judge 1'];
+    const jValue = component.getNameValue(judges);
+
+    expect(component.getNameValue).toHaveBeenCalled();
+    expect(dValue).toBe('Defendant 1');
+    expect(jValue).toBe('Judge 1');
+  });
```

Comment:
If these 2 tests are for the `getNameValue` method I think it's worth wrapping then in a `describe` block to logically separate them.

The same goes for the other sets of tests.